packages/stateful/widgets/widgets/VestingPayments/components/VestingPaymentCard/VestingPaymentCard.tsx

```diff
@@ -495,31 +495,8 @@ export const VestingPaymentCard = ({
                 decimals={token.decimals}
                 symbol={token.symbol}
               />
-
-              {!isNativeIbcUsdc(token.chainId, token.denomOrAddress) &&
-                (lazyInfo.loading || lazyInfo.data.usdUnitPrice) && (
-                  <div className="flex flex-row items-center gap-1">
-                    <TokenAmountDisplay
-                      amount={
-                        lazyInfo.loading
-                          ? { loading: true }
-                          : claimedAmount * lazyInfo.data.usdUnitPrice!.amount
-                      }
-                      dateFetched={
-                        lazyInfo.loading
-                          ? undefined
-                          : lazyInfo.data.usdUnitPrice!.timestamp
-                      }
-                      estimatedUsdValue
-                    />
-
-                    <TooltipInfoIcon
-                      size="xs"
-                      title={t('info.estimatedUsdValueTooltip')}
-                    />
-                  </div>
-                )}
             </div>
+            
```

Comment:
```suggestion
```

the linter is uspet because there's an extra line here. removing this line should satisfy it

Reply:
Thanks. I fixed this